scripts/publish-docs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
You also probably can use any POSIX shell in the shebang other than bash.

Either way :+1:

On 7 April 2016 01:24:33 CEST, Sebastian Kippe <notifications@github.com> wrote:
>> @@ -0,0 +1,14 @@
>> +#!/bin/bash
>
>Seems to work well. Thanks a lot!
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/remotestorage/remotestorage.js/pull/925/files/627754e2f826033ad0e6737c38d95b7264875a9b#r58798898

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.